src/components/seekbar/seekbar.tsx

```diff
@@ -28,7 +28,8 @@ const mapStateToProps = state => ({
   hidePreview: state.seekbar.hidePreview,
   hideTimeBubble: state.seekbar.hideTimeBubble,
   segments: state.seekbar.segments,
-  seekbarClasses: state.seekbar.seekbarClasses
+  seekbarClasses: state.seekbar.seekbarClasses,
+  isPreventSeek: state.seekbar.isPreventSeek
```

Comment:
so seekbar will be aware of preventSeek even though it's in a plugin :/
how complicated would it be to override the default seekbar by the plugin like we do in related for example ?

Reply:
actually, I don't want to overwrite the seekbar, i want ivq (and maybe other plugins in the future) to have access to this value and using redux seems like the best way